appointment/reschedule/selfServiceAppointmentRescheduleController.js

```diff
@@ -71,7 +71,7 @@ define('selfServiceAppointmentRescheduleController', [], function() {
 			const token = $scope.token;
 			const firstName = $scope.clientData.firstName;
 			const lastName = $scope.clientData.lastName;
-			const emailAddress = $scope.clientData.email;
+			const emailAddress = $scope.clientData.email || '';
```

Comment:
If I didn't coalesce this to the empty string, it would send it back as the literal string 'undefined'--I figured it was better to use the empty string rather than the string 'undefined' haha